packages/server/src/automations/steps/sendSmtpEmail.ts

```diff
@@ -48,6 +48,35 @@ export const definition: AutomationStepSchema = {
           type: AutomationIOType.STRING,
           title: "HTML Contents",
         },
+        addInvite: {
+          type: AutomationIOType.BOOLEAN,
+          title: "Add calendar invite?",
```

Comment:
We don't generally make these questions, I think it should just be `Add calendar invite` to be consistent with other checkbox driven options in the system.

Reply:
Question mark removed.